dist/docker-images/ziti-controller/compose.test.bash

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# exec this script to test the checked-out Docker controller and router deployments
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+cleanup(){
+	docker compose --profile test down --volumes --remove-orphans
+}
+
+BASEDIR="$(cd "$(dirname "${0}")" && pwd)"
+REPOROOT="$(cd "${BASEDIR}/../../.." && pwd)"
+cd "${REPOROOT}"
+
+[[ -s ./go.work ]] && {
```

Comment:
i'm not sold on this one way or the other. seems like it might be useful to be able to build/test a docker container that uses a specific set of dependencies. that or allow someone to pass in the pre-built binary location so they can work around this sort of decision that might not make sense for them.

Reply:
I agree. I'd encountered a glitch when I first attempted to build with a workspace, but this becomes a more useful kind of local smoketest if that can be solved. I'll peck at it.